src/server/controllers/index.js

```diff
@@ -23,7 +23,8 @@ Router.get('/overview', getCount.get);
 // Participants Routes
 Router.get('/participants', participants.get);
 Router.get('/participant/:id', participants.getDetails);
-Router.post('/participants/search/name', participants.searchByName);
+Router.post('/participants/search/forename', participants.searchByForeName);
+Router.post('/participants/search/surname', participants.searchBySurName);
```

Comment:
@mohannadhanafi I think `get`, instead of `post`, would be the most accurate verb to use for these endpoints since they're just reading data, not creating / changing anything.